src/services/index.js

```diff
@@ -24,6 +34,10 @@ export default async function fetchMovies(url, options) {
   return movies;
 }
 
+export const setInStorage = (key, value) => localStorage.setItem(key, JSON.stringify(value));
```

Comment:
i think we shouldn't export this two functions, the handling of the localstorage will be better contained here, let's centralize the use of it to have more domain under our stored data